timeless/reservations/views.py

```diff
@@ -92,45 +100,55 @@ def get(self, company_id):
         return jsonify(reservations_json)
 
 
-@bp.route("/")
-def list_reservations(reservations):
+@bp.route("/create", methods=("GET", "POST"))
```

Comment:
@ebercamargo We have a generic view  https://github.com/timelesslounge/timelessis/blob/master/timeless/views.py#L242 
let's use it